common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -353,9 +292,11 @@
                     if (this.group_id) {
                         options.group_id = this.group_id;
                     }
+                    break;
+                default:
                 }
-                _ref = this.collection.last();
-                lastThread = _ref ? _ref.get('id') : void 0;
+                ref = this.collection.last();
```

Comment:
As long as we're changing the names of these variables (`ref`, `ref1`, I saw an `m` up above) it would be good to make them more descriptive. I thought we had an eslint rule that prevents single-character and non-descriptive variable names, in fact.